p2panda-rs/src/document/mod.rs

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+//! A Document is a resolvable data type which is made up of a linked graph of operations. Documents MUST have a single root ‘CREATE’
+//! operation. All other operations which mutate the initial data are inserted from this point and, due to the nature of operations,
+//! connect together to form a directed acyclic graph.
```

Comment:
I've noticed that Rust Docs usually show the first paragraph of a Docstring in the overview which would be rather long in that case. Just give it a new paragraph after the first sentence, than it should look better :sparkles: 

General note on this text: Its great but also sounds very "speccy", is there a way to reformulate it to address developers "using" this API?